.travis.yml

```diff
@@ -41,6 +41,10 @@ install:
       fi
   - pip install -U tox twine wheel codecov
 
+addons:
+  hosts:
+    - xn--pxaix.localhost
```

Comment:
does it mean tests will fail locally if we don't add the same resolution to local /etc/hosts?